diff -urNp -x '*.orig' logcheck.org/etc/logcheck.logfiles logcheck/etc/logcheck.logfiles
--- logcheck.org/etc/logcheck.logfiles	2015-12-10 19:14:10.000000000 +0100
+++ logcheck/etc/logcheck.logfiles	2024-04-01 17:57:58.254695929 +0200
@@ -1,4 +1,4 @@
 # these files will be checked by logcheck
 # This has been tuned towards a default syslog install
 /var/log/syslog
-/var/log/auth.log
+/var/log/secure
